.github/workflows/testing-all-oses.yml

```diff
@@ -29,3 +29,21 @@ jobs:
     - name: Run tests
       timeout-minutes: 40
       run: pixi run -e dev env QT_QPA_PLATFORM=offscreen pytest -v -n logical --durations=20 --cov=mslib tests
+    - run: pixi run -e dev coverage xml
+    - name: Send coverage to Coveralls (parallel)
+      uses: coverallsapp/github-action@648a8eb78e6d50909eff900e4ec85cab4524a45b # v2.3.6
+      with:
+        debug: true
+        parallel: true
```

Comment:
do we need to take care for a combine too?

https://coverage.readthedocs.io/en/latest/cmd.html#combining-data-files-coverage-combine

Reply:
I don't think so, coveralls should be responsible to visualize the coverage from the different test runs in an appropriate way. I'd say we should wait and see how it works out when the macOS tests get reliable, and if there is anything we would like to have changed about how coverage is reported then.